kore/src/Kore/Step/Simplification/Predicate.hs

```diff
@@ -373,3 +397,62 @@ simplifyCeil ::
     simplifier NormalForm
 simplifyCeil sideCondition =
     Ceil.simplify sideCondition >=> return . MultiOr.map (from @(Condition _))
+
+simplifyExists ::
+    forall simplifier.
+    Monad simplifier =>
+    SideCondition RewritingVariableName ->
+    Exists () RewritingVariableName NormalForm ->
+    simplifier NormalForm
+simplifyExists _ = \exists@Exists{existsChild} ->
```

Comment:
I'd leave it for now. I think it keeps the same format as the other `simplify..` functions and it lets the implementation (instead of the caller) decide what to do with the `SideCondition`.